opengate/managers.py

```diff
@@ -452,6 +452,14 @@ class PhysicsManager(GateObject):
                 "is_input_file": True,
             },
         ),
+        "surface_properties_file": (
+            Path(os.path.dirname(__file__)) / "data" / "SurfaceProperties.xml",
+            {
+                "doc": "Path to the xml file containing the surface material properties to be used by G4LogicalBorderSurface. "
+                "Default: file shipped with Gate.",
```

Comment:
f"Default shipped with GATE located in {Path(os.path.dirname(__file__)) / "data" / "SurfaceProperties.xml"}"